docs/pywbemcli/cmdshelp.rst

```diff
@@ -1547,6 +1548,152 @@ Help text for ``pywbemcli instance shrub`` (see :ref:`instance shrub command`):
       -h, --help                      Show this help message.
 
 
+.. _`pywbemcli namespace --help`:
+
+pywbemcli namespace --help
+--------------------------
+
+
+
+Help text for ``pywbemcli namespace`` (see :ref:`namespace command group`):
+
+
+::
+
+    Usage: pywbemcli [GENERAL-OPTIONS] namespace COMMAND [ARGS] [COMMAND-OPTIONS]
+
+      Command group for CIM namespaces.
+
+      This command group defines commands to create, delete and list namespaces in a WBEM server.
```

Comment:
I note that we use the phrase CIM namespace once and thereafter it is always namespace.  Should we note 1. what is CIM namspace 2) namespace refers to CIM namespace.?

Reply:
We are doing that also for other CIM objects. I don't think we need that level of precision in the docs.
What do you think?

Reply:
OK with me.